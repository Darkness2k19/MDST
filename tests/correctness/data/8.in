10 15
1 2 3 4 5 6 7 8 9 10
1 2
3 1
4 2
5 1
6 1
7 6
8 2
9 1
10 7
1 8
1 7
2 7
3 10
4 6
4 9
