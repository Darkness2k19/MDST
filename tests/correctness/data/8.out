10 9
10 8 5 7 3 2 1 6 9 4 
8 2
5 1
4 2
7 6
3 1
10 7
4 6
1 2
4 9